### PR TITLE
Add resizerootfs scripts and add to one board as example

### PR DESCRIPTION
**Pull request by dbast**
Following up to the discussion and information in https://github.com/mkaczanowski/packer-builder-arm/issues/19

Adding as an example as this could be helpful for almost all boards and distributions.

Tested with rpi1b here.

Pinging @bcomnes who also participated in the related discussion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +127,7 @@ To execute command within chroot environment you should use chroot communicator:
  }
 ]
 ```
+For an extended example that does provisioning via script including provisioning of a resize-root-fs systemd service see [here](./boards/raspberry-pi/archlinuxarm.json).
```
Comment (mkaczanowski) [inline README.md:130]: I would rewrite this a bit :) WDYT?

```
This plugin doesn't resize partitions on the base image. However, you can easily expand partition size at the boot time with a systemd service. [Here](./boards/raspberry-pi/archlinuxarm.json) you can find real-life example, where a raspberry pi root-fs partition expands to all available space on sdcard.
```

Comment (mkaczanowski) [reply README.md:130]: This plugin doesn't resize partitions on the base image. However, you can easily expand partition size at the boot time with a systemd service. [Here](./boards/raspberry-pi/archlinuxarm.json) you can find real-life example, where a raspberry pi root-fs partition expands to all available space on sdcard.

**Comment by mkaczanowski** [comment]
other than minor comment, LGTM. Thanks!

**Comment by bcomnes** [comment]
Thanks again @dbast!

**Pull request merged**